src/Integration/Helpers/VisualStudioHelpers.cs

```diff
@@ -26,7 +26,20 @@ namespace SonarLint.VisualStudio.Integration
 {
     public static class VisualStudioHelpers
     {
-        public static readonly string VisualStudioVersion = GetVisualStudioVersion();
+        private static readonly Version visualStudio2015Update3Version = Version.Parse("14.0.25420.00");
+
+        public static string VisualStudioVersion
+        {
+            get;
+            /* for testing purpose only */ internal set;
```

Comment:
Ideally I should have created a service instead of this static class and then asked for the service in the right locations but I was lazy.